GTViewController/Classes/Custom Classes/UISnuffleButton.m

```diff
@@ -445,15 +454,19 @@ - (void)touchesEnded:(NSSet *)touches withEvent:(UIEvent *)event {
 				
 			}
 			
-            [self.tapEvents enumerateObjectsUsingBlock:^(NSString *eventName, NSUInteger idx, BOOL *stop) {
-                if (eventName) {
-                    NSString *trimmedEventName = [eventName stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]];
-                    
-                    [[NSNotificationCenter defaultCenter] postNotificationName:UISnuffleButtonNotificationButtonTapEvent
-                                                                        object:self
-                                                                      userInfo:@{UISnuffleButtonNotificationButtonTapEventKeyEventName: trimmedEventName}];
-                }
-            }];
+            if (self.forceValidation && self.tapDelegate && [self.tapDelegate respondsToSelector:@selector(didReceiveTapOnPositiveButton:)]) {
```

Comment:
Nothing happens, but not in the way you mean it.  If the new selector implements `didReceiveTapOnPositiveButton` then the outcome of that method happens.  As it stands no other delegates implement that, so "nothing happens" meaning the else branch occurs.